### ISSUE TITLE
Problem with shapes for input

### ISSUE THREAD

**Issue by flixmk** [qubvel/segmentation_models.pytorch#617]
Hi,

I currently try to get the UNet up and running and i have the following problem:

My data is in the shape of [batch_size, 3, 800, 275].
This already seems to cause problems. A size of [batch_size, 3, 64, 64] doesnt cause problems at all.

Any reason for thisor am i making something wrong?
The 
`----> 6 mask = model(torch.ones([8, 3, 800, 275]).to(device))
      7 print(mask.shape)
      8 

5 frames
[/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py](https://localhost:8080/#) in _call_impl(self, *input, **kwargs)
   1108         if not (self._backward_hooks or self._forward_hooks or self._forward_pre_hooks or _global_backward_hooks
   1109                 or _global_forward_hooks or _global_forward_pre_hooks):
-> 1110             return forward_call(*input, **kwargs)
   1111         # Do not call functions when jit is used
   1112         full_backward_hooks, non_full_backward_hooks = [], []

[/usr/local/lib/python3.7/dist-packages/segmentation_models_pytorch/base/model.py](https://localhost:8080/#) in forward(self, x)
     14         """Sequentially pass `x` trough model`s encoder, decoder and heads"""
     15         features = self.encoder(x)
---> 16         decoder_output = self.decoder(*features)
     17 
     18         masks = self.segmentation_head(decoder_output)

[/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py](https://localhost:8080/#) in _call_impl(self, *input, **kwargs)
   1108         if not (self._backward_hooks or self._forward_hooks or self._forward_pre_hooks or _global_backward_hooks
   1109                 or _global_forward_hooks or _global_forward_pre_hooks):
-> 1110             return forward_call(*input, **kwargs)
   1111         # Do not call functions when jit is used
   1112         full_backward_hooks, non_full_backward_hooks = [], []

[/usr/local/lib/python3.7/dist-packages/segmentation_models_pytorch/unet/decoder.py](https://localhost:8080/#) in forward(self, *features)
    117         for i, decoder_block in enumerate(self.blocks):
    118             skip = skips[i] if i < len(skips) else None
--> 119             x = decoder_block(x, skip)
    120 
    121         return x

[/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py](https://localhost:8080/#) in _call_impl(self, *input, **kwargs)
   1108         if not (self._backward_hooks or self._forward_hooks or self._forward_pre_hooks or _global_backward_hooks
   1109                 or _global_forward_hooks or _global_forward_pre_hooks):
-> 1110             return forward_call(*input, **kwargs)
   1111         # Do not call functions when jit is used
   1112         full_backward_hooks, non_full_backward_hooks = [], []

[/usr/local/lib/python3.7/dist-packages/segmentation_models_pytorch/unet/decoder.py](https://localhost:8080/#) in forward(self, x, skip)
     36         x = F.interpolate(x, scale_factor=2, mode="nearest")
     37         if skip is not None:
---> 38             x = torch.cat([x, skip], dim=1)
     39             x = self.attention1(x)
     40         x = self.conv1(x)

RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 36 but got size 35 for tensor number 1 in the list.`

This is the complete error message. Maybe someone can help  me out there.

**Comment by rocketrose99** [comment]
TLDR: your data resolution needs to be divisible by 32. 

Actual explanation: I had a similar issue. I poked around a while in [this example](https://github.com/qubvel/segmentation_models.pytorch/blob/master/examples/cars%20segmentation%20(camvid).ipynb) and realized they even augmented the validation data to make resolution divisible by 32. 

I followed the example's process to augment all my data (training and val) to have resolutions that were divisible by 32, and that fixed the issue for me. 

No idea why, just started trying to implement this on my data today.
